config.yaml

```
@@ -100,52 +98,42 @@ params:
         url: /license/
 
   section1:
-    title: Making sense of advice
-    subtitle: the trouble with "best practices" is they never seem to work
+    title: Why this exists
+    subtitle: because knowing what to do matters less than knowing when to do it
     text: >
-      We tend to learn a great deal about a wide variety of topics. Yet, from time to time, we are stumped by how elegantly or easily
-      someone solves a certain problem. You are left wondering why you have not been tackling similar problems in the same way. It could have saved you
-      vast amounts of frustration if only you had known earlier. <br />
-      Why did you not know? Why did you not see the solution? Why did you not apply the same technique? 
+      You've probably encountered an elegant solution to a problem and thought: "Why didn't I see that sooner?" The reason is simpler than you'd expect—you didn't know it existed. Or maybe someone mentioned it, but the context got lost in translation.
```

```suggestion
      You've probably encountered an elegant solution to a problem and thought: "Why didn't I see that sooner?" The reason is simpler than you'd expect: you simply didn't know it was an option. Maybe someone mentioned the approach once, but the context got lost in translation.
```

Avoid using em-dashes in descriptions.